pkg/docker/schema/schema.json

```diff
@@ -109,6 +109,138 @@
             }
           },
           "additionalProperties": false
+        },
+        "push": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "tag": {
+              "type": "string"
+            },
+            "arguments": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "flags": {
+              "type": "object",
+              "additionalProperties": {
+                "type": ["null","boolean","number","string"]
+              }
+            },
+            "outputs": {
+              "$ref": "#/definitions/outputs"
+            }
+          },
+          "additionalProperties": false
+        },
+        "run": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "image": {
+              "type": "string"
+            },
+            "command": {
+              "type": "string"
+            },
+            "detach": {
+              "type": "boolean"
+            },
+            "ports": {
```

Comment:
We can fill in the type further for ports in a follow-up as well. Can you add an issue to work on the schema command more? It's finicky and wasn't part of the original scope of this work so there's no need to try to do it in this PR.